src/components/App.tsx

```diff
@@ -23,12 +24,24 @@ export default function App() {
             <DefaultMain>
                 <header class="w-full flex justify-between items-center mt-4 mb-2">
                     <div class="flex items-center gap-2">
-                        <img
-                            id="mutiny-logo"
-                            src={pixelLogo}
-                            class="h-[25px] w-[75px]"
-                            alt="Mutiny logo"
-                        />
+                        <Switch>
+                            <Match when={state.subscription_timestamp}>
```

Comment:
I think there should be a function checking whether `state.subscription_timestamp` is not expired yet.

We can do a ~3 day grace period thing in a future PR later this month. But until then, all of the checks on this valud should also check whether or not it is in the past.